Makefile

```diff
@@ -177,4 +181,22 @@ manpage:
 	  --variable=section:1 \
 	  --variable=date:'June 6, 2019' \
 	  --variable=footer:'version 0.9.1' \
-	  --variable=header:'qpc-tools'
\ No newline at end of file
+	  --variable=header:'qpc-tools'
+
+# Install python egg
+OMIT_PATTERNS = */test*.py,*/.virtualenvs/*.py,*/virtualenvs/*.py,.tox/*.py
+
+install:
+	$(PYTHON) setup.py build -f
+	$(PYTHON) setup.py install -f
+
+lint:
+	tox -e lint
+
+test:
```

Comment:
We need to add `test` & `test-coverage` to the makefile help. 

This next part is optional, I just wanted to put it out there: 
We may want to also think about changing the names for these since we already have:
```
  test-all                                       Launch VMs for all supported Operating Systems
  test-rhel-6                                    Launch the RHEL 6 VM for testing
  test-rhel-7                                    Launch the RHEL 7 VM for testing
  test-rhel-8                                    Launch the RHEL 8 VM for testing
  test-centos-6                                  Launch the CentOS 6 VM for testing
  test-centos-7                                  Launch the CentOS 7 VM for testing
```
for bringing up the vagrant machines.

so maybe like:
`test-cli`
`test-cli-coverage`

Reply:
👍